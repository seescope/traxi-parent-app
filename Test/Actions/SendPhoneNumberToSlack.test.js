import sendPhoneNumberToSlack from '../../App/Actions/SendPhoneNumberToSlack';

global.Promise = require.requireActual('promise');

it('sends a phone number to Slack', () => {
  const phoneNumber = '+61401633346';

  const mockDispatch = jest.fn();
  fetch = jest.fn(() => Promise.resolve());

  return sendPhoneNumberToSlack(phoneNumber)().then(() => {
    expect(mockDispatch.mock.calls).toMatchSnapshot();
    expect(fetch.mock.calls).toMatchSnapshot();
  });
});
