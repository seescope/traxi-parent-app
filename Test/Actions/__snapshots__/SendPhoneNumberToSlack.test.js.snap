exports[`test sends a phone number to Slack 1`] = `Array []`;

exports[`test sends a phone number to Slack 2`] = `
Array [
  Array [
    "+61401633346",
  ],
]
`;
